scripts/windows/msys2_prepare.bat

```diff
@@ -1,4 +1,4 @@
-@echo on
+@echo off
```

Comment:
I found this to be `on` useful when debugging build issues.

Reply:
Actually it was `on` but all commands (except few) were implicitly 'muted', so it didn't change the output. I turned it `on` to not change the behavior, but removed `@` so it would be easier to just change this line to toggle the echo.

It could be made to be controlled from the command line though

Reply:
I plan to revise those scripts again, so they are more robust and compatible with the newly introduced CircleCI windows builds, but I had hard time to debug those, and are currently postponed. So may be I will  include optional flag to turn echo on with it